### PR TITLE
docs(datepicker): demo for supported formats, based on moment.js(1:1) #3590

### LINKED ISSUE #3590: docs(datepicker): add demo for supported formats, based on moment.js(1:1)
Add demo for supported formats, based on moment.js(1:1) (related to #3340)

### PR DESCRIPTION
**Pull request by IraErshova**
Extend demo Custom date format in datepicker with several moment.js formats

Close #3590

# PR Checklist
Before creating new PR, please take a look at checklist below to make sure that you've done everything that needs to be done before we can merge it.

 - [x] read and followed the [CONTRIBUTING.md](https://github.com/valor-software/ngx-bootstrap/blob/development/CONTRIBUTING.md) guide.
 - [x] built and tested the changes locally.
 - [x] added/updated API documentation.
 - [x] added/updated demos.

### CODE REVIEW

File: demo/src/app/components/+datepicker/datepicker-section.list.ts [hunk 1]
```diff
@@ -59,7 +59,10 @@ export const demoComponentContent: ContentSection[] = [
         html: require('!!raw-loader?lang=markup!./demos/custom-format/date-picker-custom-format.html'),
         description: `
           <p>You can easily change the date format by specifying the <code>dateInputFormat</code> in <code>[bsConfig]</code></p>
-          <p>The following example shows how to use a datepicker having <code>YYYY-MM-DD</code> date format inside a form:</p>
+          <p>To set your own date format you can use variety of formats provided by
+          <a href="https://momentjs.com/docs/#/displaying/format/" target="_blank">moment.js</a></p>
+          <p>The following examples shows how to use a datepicker having <code>YYYY-MM-DD</code>, <code>MM/DD/YYYY</code>,
```
Comment (IlyaSurmay) [inline demo/src/app/components/+datepicker/datepicker-section.list.ts]: shows -> show

**Review by IlyaSurmay — CHANGES_REQUESTED** [review]
Be consistent in naming of files. If demo folder  is `custom-format`, then html and ts in it should have the same name. This is essential for generating plunkr and stackblitz demos.

**Review by IlyaSurmay — DISMISSED** [review]

**Comment by YevheniiaMazur** [comment]
Letter a marked as code, but it's not. And it will be better, if description of date format couldn't be splitting, and places at the same line
smallissue
`
Otherwise looks good

**Comment by YevheniiaMazur** [comment]
Now checking prs is blocked by package issue.

**Comment by YevheniiaMazur** [comment]
Tested, looks good

**Pull request merged**